lib/pages/forward/forward_view.dart

```diff
@@ -35,38 +34,29 @@ class ForwardView extends StatelessWidget {
           textEditingController: controller.searchTextEditingController,
         ),
       ),
-      body: PopScope(
-        canPop: false,
-        onPopInvoked: (didPop) async {
-          if (PlatformInfos.isAndroid) {
-            controller.popScreen();
-          }
-        },
```

Comment:
how about the case swift to back?

Reply:
works as expected. Please see the videos in description